docs/essentials/preferences.md

```diff
@@ -47,6 +47,32 @@ Preferences.Clear();
 
 In addition to these methods each take in an optional `sharedName` that can be used to create additional containers for preference. Read the platform implementation specifics below.
 
+## Data Binding
+
+You can also use a Preference in a view model property for use with XAML data binding. 
+
+#### View model
```

Comment:
Heading levels should cascade, so this should be H3 ( `###`). also, add a blank line after the heading.